sdk/core/src/http_client/reqwest.rs

```diff
@@ -24,9 +26,18 @@ impl HttpClient for ::reqwest::Client {
 
         let reqwest_request = match body {
             Body::Bytes(bytes) => req.body(bytes).build(),
+
+            #[cfg(not(target_arch = "wasm32"))]
             Body::SeekableStream(seekable_stream) => req
                 .body(::reqwest::Body::wrap_stream(seekable_stream))
                 .build(),
+
+            // Cannot currently implement `Body::SeekableStream` for WASM
+            // because `reqwest::Body::wrap_stream()` is not implemented for WASM.
+            #[cfg(target_arch = "wasm32")]
+            Body::SeekableStream(_seekable_stream) => {
+                todo!("Body::SeekableStream is not currently supported for WASM")
```

Comment:
Instead of using `todo!` here, can the `SeekableStream` variant of `Body` be only defined in via `not(target_arch = "wasm32")`?

Failing at compile-time is much less surprising than failing at runtime due to a missing platform feature.

Reply:
Yep - fixed.  I obviously had to add `wasm32` conditionals to all the places where `SeekableStream` was used.